tests/network/localnet/test_default_bridge.py

```diff
@@ -33,3 +40,22 @@ def test_connectivity_post_migration_between_localnet_vms(migrated_localnet_vm,
         port=8888,
     ) as (client, server):
         assert is_tcp_connection(server=server, client=client)
+
+
+@pytest.mark.ipv4
+@pytest.mark.single_nic
+@pytest.mark.s390x
+@pytest.mark.usefixtures("nncp_localnet")
+@pytest.mark.polarion("CNV-12363")
+def test_vmi_reports_ip_on_secondary_interface_without_vlan(
+    vm_localnet_1,
+    vm_localnet_1_secondary_ip,
```

Comment:
This info could have been fetched from the spec itself.
Said that, this could be done in a follow up.